pkg/libinit/root_linux.go

```diff
@@ -284,3 +295,71 @@ func InstallModules(pattern string, exclude map[string]bool) error {
 
 	return nil
 }
+
+func readModules(f *os.File) []string {
+	r := bufio.NewReader(f)
+	modules := []string{}
+	for {
+		if i, err := r.ReadString('\n'); err == nil {
+			if strings.HasPrefix(i, "#") {
+				continue
+			} else {
+				modules = append(modules, strings.TrimSuffix(i, "\n"))
+			}
+		} else {
+			break
+		}
+	}
+	return modules
+}
+
+// InstallModules installs kernel modules from .conf files in /etc/modules-load.d/
+func InstallModulesFromModulesLoad(pattern string, exclude map[string]bool) error {
+	files, err := filepath.Glob(pattern)
+	if err != nil {
+		return err
+	}
+
+	for _, filename := range files {
+		f, err := os.Open(filename)
+		if err != nil {
+			log.Printf("installModulesFromModulesLoad: can't open %q: %v", filename, err)
```

Comment:
The log line prints "installModulesFromModulesLoad" but the function name is "InstallModulesFromModulesLoad" (capital "I" at the start)